src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -447,5 +447,6 @@
             az acr image import -n targetRegistry --source-image sourceregistry.azurecr.io/repository:tag -t repo:tag .
         - name: Import from a registry in a different subscription from the one the target registry is in
           text: >
-            az acr image import -n yugongTest --resource-id /subscriptions/dfb63c8c-7c89-4ef8-af13-75c1d873c895/resourceGroups/yugong/providers/Microsoft.ContainerRegistry/registries/yugongeast --source-image yugongeast.azurecr.io/builder:latest -t test:test .
+            az acr image import -n targetRegistry --resource-id /subscriptions/dfb63c8c-7c89-4ef8-af13-75c1d873c895/resourceGroups/yugong/providers/Microsoft.ContainerRegistry/registries/yugongeast --source-image yugongeast.azurecr.io/builder:latest
+            -t test:test
```

Comment:
Replace your names with some common strings, also the sub id with empty guid, see other examples.